lib/glimesh/chat.ex

```diff
@@ -245,6 +264,23 @@ defmodule Glimesh.Chat do
     end
   end
 
+  def message_contains_link(chat_message) do
+    regex_string = ~r/ (?:(?:https?|ftp)
+                        :\/\/|\b(?:[a-z\d]+\.))(?:(?:[^\s()<>]+|\((?:[^\s()<>]+|(?:\([^\s()<>]+\)))
+                        ?\))+(?:\((?:[^\s()<>]+|(?:\(?:[^\s()<>]+\)))?\)|[^\s`!()\[\]{};:'".,<>?«»“”‘’]))?
+                      /xi
+
+    found_uris = flatten_list(Regex.scan(regex_string, chat_message))
```

Comment:
Combine this with hyperlink_message to have the same regex being changed at the same time if there are changes.